File: Naloga 9/naloga_9.cpp
#include <iostream>

void naloga_9 () {
    int stevilo, vsota = 0;
    std::cin >> stevilo;
    for (int stevec = 1; stevec <= stevilo; ++stevec) {
        vsota += stevec * (stevec + 1) / 2;
    }
    std::cout << vsota << std::endl;
};

int main () {
    naloga_9();
    return 0;
}

File: Naloga 9/naloga_9.py
def naloga_9():
    # deklaracija spremenljivk
    stevilka = int(input("vpisi stevilko: "))
    vsota = 0
    stevec = 0 
    # zanka
    while stevec != stevilka:
        stevec += 1  # stevec (stevec = stevec +1)
        gaus = stevec * (stevec+1) / 2  # Gausov algo.
        print(stevec, gaus, vsota)
        vsota += gaus

    print(vsota)
    return


if __name__ == '__main__':
    naloga_9()
    